tools/loc/sharepoint.js

```diff
@@ -378,10 +378,8 @@ async function getMetadata(srcPath, file) {
 
 async function copyFile(srcPath, destinationFolder, newName, isFloodgate, isFloodgateLockedFile) {
   validateConnection();
-  await createFolder(destinationFolder, isFloodgate);
```

Comment:
@Nethraasivakumar Removal of this might break localization as it calls the same copy function. 
https://github.com/adobecom/milo/blob/main/tools/loc/ui.js#L579
Was that validated with loc?

I would suggest keeping this line as-is in the milo repo but removing it from sharepoint.js that is part of the milo-fg repo as that will be floodgate only.

Reply:
@sukamat Had missed checking for the dependency of localisation on copyFile. Checked it now and found that you are right. Have reverted the code in order to include the createFolder line so that there's no discrepancy for the localisation flow. As mentioned by you, I'll remove the line when I raise a PR for the milo-fg repo.